### DISCUSSION TITLE
How to run  Xamarin.Forms - Controls Gallery app on iOS simulator

### DISCUSSION THREAD

**Discussion by Elashi** [xamarin/Xamarin.Forms#14097] [Q&A]
I added a feature to Entry and Editor controls.  I was able to test it on Android app but is there a way to test it on iOS simulator from my Windows machine without a mac ?

**Comment by EmilAlipiev** [comment]
no you cannot. you need a mac machine. Hotrestart enables you to debug directly on Iphone or Ipad but since UI testing involves release build, you will not able to also.
